src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -140,16 +158,28 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
           const onClick = () => onTabClick(index);
           return (
             <li key={index}>
-              <button
-                className={cx('iui-tab', {
-                  'iui-active': index === currentIndex,
-                })}
-                onClick={onClick}
-                role='tab'
-                aria-selected={index === currentIndex}
-              >
-                {label}
-              </button>
+              {typeof label === 'string' ? (
+                <HorizontalTab
+                  label={label}
+                  className={cx('iui-tab', {
+                    'iui-active': index === currentIndex,
+                  })}
+                  onClick={onClick}
```

Comment:
I think active tab should be changed on click. 
![horizontal-tabs](https://user-images.githubusercontent.com/83585998/125276879-69951e80-e319-11eb-8f91-7c59435fa133.gif)

Reply:
It works on my side 🤔 Maybe you need to run `yarn install` for the latest css.